### PR TITLE
IT-2398: Add JC access to Genie BPC Prod account

### PR DESCRIPTION
**Pull request by brucehoff**
Add JC access to Genie BPC prod account as part of the account set up process.

### CODE REVIEW

File: org-formation/700-aws-sso/_tasks.yaml [hunk 1]
```diff
@@ -1435,3 +1443,37 @@ SsoRecoverProdViewer:
...
+  DefaultOrganizationBindingRegion: !Ref primaryRegion
+  DefaultOrganizationBinding:
+    IncludeMasterAccount: true
+  OrganizationBindings:
+    TargetBinding:
+      Account: !Ref GenieProdAccount
+  Parameters:
+    instanceArn: !Ref instanceArn
+    principalId: !Ref genieProdViewerPlusGroup
+    permissionSetArn: !CopyValue [ !Sub '${resourcePrefix}-${appName}-viewer-permission-set-arn' ]
```
Comment (zaro0508) [inline org-formation/700-aws-sso/_tasks.yaml:1479]: you probably want viewer-plus permission set here?

File: org-formation/700-aws-sso/_tasks.yaml [hunk 2]
```diff
@@ -225,6 +225,14 @@ Parameters:
     Type: String
     Default: 'b4e86468-f091-70e2-0ea9-64cf3addcdf1'
 
+  genieProdViewerGroup:      #JC aws-genie-prod-viewers
```
Comment (zaro0508) [inline org-formation/700-aws-sso/_tasks.yaml:228]: no admins or developer role needed for this account?

**Review by zaro0508 — CHANGES_REQUESTED** [review]

File: org-formation/700-aws-sso/_tasks.yaml (hunk 1 above)
Comment (brucehoff) [reply org-formation/700-aws-sso/_tasks.yaml]: yes, thanks.

File: org-formation/700-aws-sso/_tasks.yaml (hunk 2 above)
Comment (brucehoff) [reply org-formation/700-aws-sso/_tasks.yaml:228]: Correct.  We will follow up with a new role that gives app' creators permission to manage secrets (see https://sagebionetworks.jira.com/browse/DCA-35) but not have administrative access.

**Review by zaro0508 — APPROVED** [review]

**Pull request merged**